libraries/Microsoft.Bot.Builder.LanguageGeneration/TemplateErrors.cs

```diff
@@ -10,6 +10,7 @@ namespace Microsoft.Bot.Builder.LanguageGeneration
     public class TemplateErrors
```

Comment:
>TemplateErrors [](start = 17, length = 14)

Might be worth adding a see tag to this class in the public methods where this would be a likely error.

Reply:
I change the TemplateErrors class to internal. And added below text to make sure the tests can use this class.
```
using System.Runtime.CompilerServices;

[assembly: InternalsVisibleTo("Microsoft.Bot.Builder.LanguageGeneration.Tests")]
```
Also I added a <see cref = "TemplateErrors"> tag in some places which throw LG errors.